app/uk/gov/hmrc/soletraderidentification/featureswitch/core/config/FeatureSwitchingModule.scala

```diff
@@ -25,7 +25,7 @@ import javax.inject.Singleton
 @Singleton
 class FeatureSwitchingModule extends Module with FeatureSwitchRegistry {
 
-  val switches = Seq(StubGetSaReference, DesStub, CreateTrnStub, InsightStub)
+  val switches = Seq(DesStub, CreateTrnStub, InsightStub)
```

Comment:
StubGetSaReference is being used in AppConfig. Do we really want to remove it from the list of feature switches? I think this will deactivate the stub.

Reply:
We have a controller/service/connector set up for it but the FE doesn't actually call it so whilst I was in the file I thought I'd just get it removed from the stub list so it's less confusing but happy to add it back in and get a tech debt ticket raised to remove all the unused code

Reply:
I think it is Ok to leave the list of feature switches as it is, but I'd raise a ticket to remove the code if it's not needed.

Reply:
Removed it for now. Look to update in the future